### ISSUE TITLE
Flashing text on live widget

### ISSUE THREAD

**Issue by apfritts** [Kemmey/Kemmey-TeslaWatch-Public#517]
The live widget of heart but the flashing text odd annoying. 

https://user-images.githubusercontent.com/2000021/205430382-6157888a-f81f-4398-9abb-903b2132630e.MOV

**Comment by apfritts** [comment]
Screenshot of “frozen” live widget:
![E0395F4A-9890-498E-ADF5-DD055C0C1F56](https://user-images.githubusercontent.com/2000021/205453647-8820017e-abe4-471b-be2e-3d3b77c85d9c.png)

**Comment by apfritts** [comment]
I could see myself paying $12/year for the live widget feature. If you also include the fact you can do Siri Shortcuts, you could probably fetch double that even though I don’t think those Siri Shortcuts can be behind a paywall.

**Comment by tseval** [comment]
I agree, I like the Live widget, but the flashing text is annoying.

Also, there should be some setting to enable/disable the live widget if I didn't want to use it

**Comment by Kemmey** [comment]
Yeah - the flashing text is super annoying - it's a bug introduced by apple in iOS 16.2+ - nothing I can fix, but might be able to work around. 

Settings for live activities are built-in to iOS and you have to confirm to keep first time. 

![image](https://user-images.githubusercontent.com/4688084/210857660-e939f314-0e95-47b5-a981-3f52e2c25a30.png)

**Comment by tseval** [comment]
Heh, would you look at that! I even checked this Settings page yesterday, but managed to miss that toggle even while looking straight at it 😆

Thanks! And hopefully you'll be able to fix the blinking as well. Keep up the good work, I love the app
